src/RecoveryServices/RecoveryServices.Backup.Helpers/Conversions/RecoveryPointConversions.cs

```diff
@@ -775,6 +794,12 @@ public static ServiceClientModel.RecoveryPointTierType GetServiceClientRecoveryP
                 }
             }
 
+            /*if (recoveryPoint.RecoveryPointProperties != null)
```

Comment:
Please remove commented code unless you'll need it in the future, in which case add a note of when to uncomment

Reply:
will need in future, let me add a note